### ISSUE TITLE
out_sock is not used

### ISSUE THREAD

**Issue by fb39ca4** [wiiuse/wiiuse#141]
https://github.com/wiiuse/wiiuse/blob/c9b98690349d2100889189ad903431cc903fc4ee/src/os_nix.c

The only place out_sock is used is wiiuse_os_connect and wiiuse_os_disconnect (as well as the init/cleanup functions). You'd think that out_sock should be used for wiiuse_os_write, but in_sock is used for both wiiuse_os_write and wiiuse_os_read.

**Comment by fb39ca4** [comment]
Got it. So it's necessary to have both the control and data pipes opened even though no communication is required over the control pipe?

**Comment by janoc** [comment]
Yes. It is part of the protocol. The control pipe is actually used by Wii and Wiimote for some things (not everything has been reverse engineered) but Wiiuse doesn't use it.
